js/bundle_test.go

```diff
@@ -189,13 +189,6 @@ func TestNewBundle(t *testing.T) {
 					"InvalidCompat", "es1", `export default function() {};`,
 					`invalid compatibility mode "es1". Use: "extended", "base", "experimental_enhanced"`,
 				},
-				// BigInt is not supported
```

Comment:
for reviewers: I dropped the case completely since I am unsure how useful it is. If reviewers find this relevant, we can come up with another not-implemented feature that could be used till the next time

Reply:
I don't think it is as useful now that basically compatibility mode is kind of defunct, so :+1:

In theory we can replace it with some other functionality that isn't supported ... but :shrug: